### DISCUSSION TITLE
Ability to easy enter parameters or global test templates

### DISCUSSION THREAD

**Discussion by akgroupllc** [inventree/InvenTree#8449] [General]
We are building out our database inside of the system to track a greenhouse. With that we may want to use things like "building orders" "purchasing" but for right now we are simply entering stock, and then tracking it. 
Right now, about the only way i can see doing that is with tests so one a year we go out into the green house and say this "part" is now X ft or something like that under the test, but this requires to add that test template to each item and we have a lot of items. 
It would be great to have ability to allow certain tests across all items.
Or am i missing something?  Thanks, this is a great piece of software!

**Comment by SchrodingersGat** [comment]
@akgroupllc one thing you can do is define a "template part" - https://docs.inventree.org/en/stable/part/template/ 

You can define a "TestTemplate" for this template part, and then any *variants* of that part will also inherit the test template.

**Reply by akgroupllc** [reply]
that is a very helpful method! thank you!
Wondering if doing something more globally could be a good thought to maybe add to a to-do list. 
Say option to have test template for all items? I get it wouldn't be useful to everyone but i think for some users it may be, but this solved my problem more or less thank you!
